Resources/Computing/ARCComputingElement.py

```diff
@@ -218,24 +220,23 @@ def submitJob( self, executableFile, proxy, numberOfJobs = 1 ):
         gLogger.debug("Successfully submitted job %s to CE %s" % (pilotJobReference, self.ceHost))
       else:
         message = "Failed to submit job because "
-        if (result == arc.SubmissionStatus.NOT_IMPLEMENTED ):
+        if (result.isSet(arc.SubmissionStatus.NOT_IMPLEMENTED) ):
           gLogger.warn( "%s feature not implemented on CE? (weird I know - complain to site admins" % message )
-        elif ( result == arc.SubmissionStatus.NO_SERVICES ):
+        if ( result.isSet(arc.SubmissionStatus.NO_SERVICES) ):
           gLogger.warn( "%s no services are running on CE? (open GGUS ticket to site admins" % message )
-        elif ( result == arc.SubmissionStatus.ENDPOINT_NOT_QUERIED ):
+        if ( result.isSet(arc.SubmissionStatus.ENDPOINT_NOT_QUERIED) ):
           gLogger.warn( "%s endpoint was not even queried. (network ..?)" % message )
-        elif ( result == arc.SubmissionStatus.BROKER_PLUGIN_NOT_LOADED ):
+        if ( result.isSet(arc.SubmissionStatus.BROKER_PLUGIN_NOT_LOADED) ):
           gLogger.warn( "%s BROKER_PLUGIN_NOT_LOADED : ARC library installation problem?" % message )
-        elif ( result == arc.SubmissionStatus.DESCRIPTION_NOT_SUBMITTED ):
+        if ( result.isSet(arc.SubmissionStatus.DESCRIPTION_NOT_SUBMITTED) ):
           gLogger.warn( "%s no job description was there (Should not happen, but horses can fly (in a plane))" % message )
-        elif ( result == arc.SubmissionStatus.SUBMITTER_PLUGIN_NOT_LOADED ):
+        if ( result.isSet(arc.SubmissionStatus.SUBMITTER_PLUGIN_NOT_LOADED) ):
           gLogger.warn( "%s SUBMITTER_PLUGIN_NOT_LOADED : ARC library installation problem?" % message )
-        elif ( result == arc.SubmissionStatus.AUTHENTICATION_ERROR ):
+        if ( result.isSet(arc.SubmissionStatus.AUTHENTICATION_ERROR) ):
           gLogger.warn( "%s authentication error - screwed up / expired proxy? Renew / upload pilot proxy on machine?" % message )
-        elif ( result == arc.SubmissionStatus.ERROR_FROM_ENDPOINT ):
+        if ( result.isSet(arc.SubmissionStatus.ERROR_FROM_ENDPOINT) ):
           gLogger.warn( "%s some error from the CE - ask site admins for more information ..." % message )
```

Comment:
Yes - after discussing with the ARC developer, there could be multiple issues flagged in the return value.